SpatialGDK/Source/SpatialGDK/SpatialGDK.Build.cs

```diff
@@ -43,7 +43,7 @@ public SpatialGDK(ReadOnlyTargetRules Target) : base(Target)
 
         var WorkerLibraryDir = Path.GetFullPath(Path.Combine(ModuleDirectory, "..", "..", "Binaries", "ThirdParty", "Improbable", Target.Platform.ToString()));
```

Comment:
I think we can use this to add an include path to the C headers, which should be easier now that we download the headers and the binaries separately. Not essential but would be nicer.